app/controllers/publishers/paypal_connections_controller.rb

```diff
@@ -0,0 +1,55 @@
+require "faraday"
```

Comment:
Do you need to require this? I was under the impression that autoloading took care of this.